dmake.yml

```diff
@@ -0,0 +1,26 @@
+dmake_version: 0.1
+app_name: deepomatic-client-python
+env:
+  default:
+    source: ${DEEPOMATIC_CONFIG_DIR}/dev.sh
+    variables:
+      DEEPOMATIC_APP_ID: ${TEST_CLIENTS_APP_ID}
+      DEEPOMATIC_API_KEY: ${TEST_CLIENTS_API_KEY}
+  branches:
+    master:
+      source: ${DEEPOMATIC_CONFIG_DIR}/prod.sh
+
+
+docker:
+  base_image:
+      name: deepomatic-client-python
+      root_image: ubuntu:17.10
```

Comment:
maybe more representative to test on ubuntu 16 ?

Reply:
I did not had a simple access to python 3.6 so switched to the latest Ubuntu. We can change to the LTS soon.